app/theme.js

```diff
@@ -0,0 +1,9 @@
+import { theme } from '@chakra-ui/core';
+
+export default {
+  ...theme,
+  colors: {
+    ...theme.colors,
+    brand: '#5D5E5E',
```

Comment:
It seems we are getting a warning as below to add variant
![image](https://user-images.githubusercontent.com/39694575/77643750-eca82f00-6f85-11ea-88be-f1529f86edb3.png)
change this line with the generated colors will solve the issue - https://coolors.co/
```json
brand: {
      '50': '#8ED0E9',
      '100': '#7BC8E6',
      '200': '#69C0E2',
      '300': '#56B8DF',
      '400': '#43B0DB',
      '500': '#31A9D8',
      '600': '#2D9AC5',
      '700': '#298BB1',
      '800': '#247B9E',
      '900': '#206C8A',
    },
```

Reply:
I'll do it